client/lib/parsing.rs

```diff
@@ -211,6 +212,9 @@ mod args_complex {
 
 const STANDARD_PAYMENT_ARG_NAME: &str = "amount";
 fn standard_payment(value: &str) -> Result<RuntimeArgs> {
+    if value.is_empty() {
+        return Err(Error::InvalidCLValue(value.to_string()));
+    }
```

Comment:
It would seem so:
```
    #[test]
    fn temp_test_u512() {
        let empty512 = U512::from_dec_str("").unwrap();
        assert_eq!(empty512, U512::from(0));
    }
```
passes

Reply:
OK - good catch then! :D